lib/services/s3.js

```diff
@@ -744,6 +744,9 @@ AWS.util.update(AWS.S3.prototype, {
    *   `ContentLength`, or `Tagging` must be provided as headers when sending a
    *   request. If you are using pre-signed URLs to upload from a browser and
    *   need to use these fields, see {createPresignedPost}.
+   * @note signatureVersion v4 is required for Range to be included as a signed
```

Comment:
We might need to make the statement more generic to all the parameters that not signable in V2 not just for Range. It may looks clearer if phrased like `The default signer allows altering the request by adding corresponding headers to set some parameters(e.g. Range) and these added parameters won't be validated by signature. You must use signatureVersion v4 to enforce the inclusion and exact matching of header and signed URL`